src/Microsoft.OData.Client/DataServiceQueryOfT.cs

```diff
@@ -314,14 +314,27 @@ public DataServiceQuery<TElement> Expand<TTarget>(Expression<Func<TElement, TTar
         /// <returns>A new <see cref="T:Microsoft.OData.Client.DataServiceQuery`1" /> object that has the inline count option set.</returns>
         public DataServiceQuery<TElement> IncludeTotalCount()
         {
-            MethodInfo mi = typeof(DataServiceQuery<TElement>).GetMethod("IncludeTotalCount");
+            MethodInfo mi = typeof(DataServiceQuery<TElement>).GetMethods()
+                .FirstOrDefault(m => m.Name == "IncludeTotalCount" && m.GetParameters().Length == 0);

```

Comment:
Consider using ```nameof(IncludeTotalCount)``` as opposed to the string ```"IncludeTotalCount"``` if the name were ever to change, a refactor will be caught and fixed.
[More info](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof)

Reply:
Done